src/shared/common/forms/text.jsx

```diff
@@ -5,15 +5,13 @@ class Text extends Component {
 
     constructor(props) {
```

Comment:
the whole constructor can go

Reply:
Who knew! The whole constructor has gone.